source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,23 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can watch for changes in MongoDB by using the `watch()
+<https://mongodb.github.io/node-mongodb-native/3.5/api/Collection.html#watch>`_ method. ``watch()`` opens a :manual:`Change Stream</changeStreams/>`, that allows applications to watch for changes to data and react to those changes. After opening a ``Change Stream``, you can process the change events using the `changeStream.next()
```

Comment:
Could you use the `node-api` role here and the other locations in which you're linking to the Node API docs to make it easier to update? See:
https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L534 for the role info.

Reply:
Fixed